requirements.txt

```diff
@@ -4,12 +4,14 @@ Pypubsub==4.0.3
 configparser==5.0.1
 h5py==2.10.0
 imageio==2.9.0
+mido==1.2.10
 nibabel==3.2.1
 numpy==1.20.1
 plaidml-keras==0.7.0
 psutil==5.8.0
 pyserial==3.5
 python-gdcm==3.0.8.1
+python-rtmidi==1.4.9
```

Comment:
```
 python-rtmidi[navigator]==1.4.9
```

Or create a new requirements file, something like `navigator_requirements.txt"

Reply:
Thanks, done (see my other comment in this PR).